k8s/manifests/charts/ck-loadbalancer/templates/metallb/lb-ip-pool.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (gt (len .Values.ipPool.cidrs) 0) }}
+{{- if (eq .Values.driver "metallb") }}
+
+apiVersion: "metallb.io/v1beta1"
+kind: IPAddressPool
+metadata:
+  name: {{ include "ck-loadbalancer.fullname" . }}
+  labels:
+    {{- include "ck-loadbalancer.labels" . | nindent 4 }}
+spec:
+  addresses:
+    {{- range .Values.ipPool.cidrs }}
+    - {{ . }}
+    {{- end }}
```

Comment:
I learned the hard-way on this.  The valid formats are pretty specific here.  this can be any of these:

`a.b.c.d/<mask>`
`aaaa:bbbb:cccc:dddd::/<mask>`
`a.b.c.d-e.f.g.h`

but definitely not `a.b.c.d`.  The metallb validating webhook will deny you with a 403 Forbidden when trying to apply.  Nothing more confusing than getting a 403 when that's not quite the right return code for invalid format.

Reply:
Good to know, thx for the comment